packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -314,7 +314,16 @@ export class StripeWebhookHandler extends StripeHandler {
       );
       return;
     }
-    await this.paypalHelper.issueRefund(invoice, transactionId);
+    try {
+      await this.paypalHelper.issueRefund(invoice, transactionId);
+    } catch (error) {
+      if (error instanceof RefusedError) {
+        await this.stripeHelper.updateInvoiceWithPaypalRefundReason(
+          invoice,
+          error.longMessage
+        );
+      }
```

Comment:
Did we want to catch all refused errors, or only the two with the particular long messages mentioned in the ticket?

> The specific errors in this case can be determined by looking at the long message returned in the error.
> There are two to catch in this case:
> * 'This transaction already has a chargeback filed'
> * 'You are over the time limit to perform a refund on this transaction'

Reply:
All refused errors, this will include the long message on the invoice, so the support person can identify why it didn't work as well.